CMakeLists.txt

```diff
@@ -73,7 +73,7 @@ find_library(rt_lib rt)
 
 find_package(Python 3.6.8 REQUIRED COMPONENTS Interpreter Development)
 find_package(Threads REQUIRED)
-find_package(Proton 0.34.0 REQUIRED COMPONENTS Core Proactor)
+find_package(Proton 0.37.0 REQUIRED COMPONENTS Core Proactor Tls)
```

Comment:
I'm opening https://issues.apache.org/jira/browse/PROTON-2532, CMake find_package will fail unhelpfully when user asks for component that is not provided

See the RPM CI job failure to see how unhelpful the failure is.